boards/arm/w5500_evb_pico/w5500_evb_pico.dts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
+ * Copyright (c) 2023 Ian Wakely
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <freq.h>
+
+#include <rpi_pico/rp2040.dtsi>
+#include "w5500_evb_pico-pinctrl.dtsi"
+#include <zephyr/dt-bindings/pwm/pwm.h>
+
+#include <zephyr/dt-bindings/i2c/i2c.h>
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,flash-controller = &ssi;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
```

Comment:
Is there any objections to have the serial console at USB like `wio_terminal` board?

Reply:
I just tested this patch + net/cloud/tagoio application and USB CDC + Ethernet works like a charm.

Reply:
The original Raspberry Pi Pico board configuration has the console set to be uart0. Seeing as this board is essentially "a Pico with an ethernet jack glued to the side", I wasn't sure if I wanted stray far away from that.

Reply:
Personally, I believe the board configs should be rather minimal sticking to what the physical hardware has, and bring in the least amount of code possible. This gives the application developer less work when they want to override functionality to remove what they aren't using.

Reply:
IMHO the drawback of not having the console in the USB is:

1- The user need to solder the header pins
2- Have an external USB<->Serial adapter
3- Maybe user change the code to enable CDC-ACM and have the console to avoid [1-2].

With the Serial Console in the USB by default it is easy to anyone validate the majority (if not all) examples and tests. Otherwise the user always have a lot of extra work with direct no benefit. Since the board came with 16MB of flash I don't see why not have it. If the user really need to use the serial port then he change what is needed because if their own use case for that.

The final decision is yours but I hope you could consider the change : )

Reply:
Unless @anangl @yonsch @soburi or @nordicjm have a strong opinion on why it should be different than the original pico, I think I'm going to leave it as is. Sorry @nandojve .

Reply:
I agree it is good to keep this configuration the same as rpi_pico.